### PR TITLE
Remove default value of repetition period from Repeat

### PR DESCRIPTION
**Pull request by keriksson-rosenqvist**

### CODE REVIEW

**Review by jfriel-oqc — APPROVED** [review]

File: src/QAT/qat/purr/compiler/instructions.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ class Repeat(Instruction):
     def __init__(self, repeat_count, repetition_period=None):
         super().__init__()
         self.repeat_count = repeat_count
-        self.repetition_period = repetition_period or 100e-6
```
Comment (hamidelmaazouz) [inline src/QAT/qat/purr/compiler/instructions.py:124]: Are there any tests for this change ?

**Comment by hamidelmaazouz** [comment]
@bryn-bell This PR is actually duplicate of https://github.com/oqc-community/qat/pull/80. Keep only one of them

**Review by hamidelmaazouz — CHANGES_REQUESTED** [review]

**Review by hamidelmaazouz — CHANGES_REQUESTED** [review]
@bryn-bell This PR is actually duplicate of https://github.com/oqc-community/qat/pull/80. Keep only one of them

**Comment by keriksson-rosenqvist** [comment]
> @bryn-bell This PR is actually duplicate of #80. Keep only one of them

@hamidelmaazouz No these are aiming at `main` and `develop` respectively. They are both required. This is the standard process as discussed.

File: src/QAT/qat/purr/compiler/instructions.py (hunk 1 above)
Comment (keriksson-rosenqvist) [reply src/QAT/qat/purr/compiler/instructions.py:124]: @hamidelmaazouz Tests added

**Review by hamidelmaazouz — APPROVED** [review]

**Pull request merged**